### PR TITLE
SDIL 811

### PR DESCRIPTION
**Pull request by nkaraolis**
We now have some basic auth integration which checks if the user has a UTR or not. This logic needs to be clarified later on but we we have the main bulk of the auth config now. 
Also refactored the directory structure to remove empty folders and the acceptance tests

### CODE REVIEW

File: project/FrontendBuild.scala [hunk 1]
```diff
@@ -3,42 +3,45 @@ import play.routes.compiler.StaticRoutesGenerator
 import play.sbt.PlayImport._
 import play.sbt.routes.RoutesKeys.routesGenerator
 import sbt.Keys._
-import sbt.Tests.{SubProcess, Group}
 import sbt._
-import uk.gov.hmrc.SbtAutoBuildPlugin
-import uk.gov.hmrc._
+import uk.gov.hmrc.DefaultBuildSettings._
+import uk.gov.hmrc.{SbtAutoBuildPlugin, _}
```
Comment (hmrclt) [inline project/FrontendBuild.scala]: ??

File: test/controllers/HelloWorldControllerSpec.scala [hunk 2]
```diff
@@ -14,48 +14,68 @@
...
+
+      status(result) mustBe OK
+      contentAsString(result) must include("Return result is: false")
+    }
+
+    "return Status: 200 Message: Return result is: true for successful auth with utr" in {
+      sdilAuthMock(userWithUtr)
+
+      val request = FakeRequest("GET", "/auth-test")
+
```
Comment (hmrclt) [inline test/controllers/HelloWorldControllerSpec.scala]: so much whitespace needed?

File: test/controllers/package.scala [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+package object controllers extends MockitoSugar {
+
+  val mockAuthConnector: AuthConnector = mock[AuthConnector]
+
+  val userWithUtr: Future[Option[String]] = Future successful Some("UTR")
+  val userNoUtr: Future[Option[String]] = Future successful None
+
+  def sdilAuthMock(returnValue: Future[Option[String]]): OngoingStubbing[Future[Option[String]]] =
+    when(mockAuthConnector.authorise(any(), any[Retrieval[Option[String]]]())(any(), any[ExecutionContext])).thenReturn(returnValue)
```
Comment (hmrclt) [inline test/controllers/package.scala]: line too long

File: test/controllers/HelloWorldControllerSpec.scala [hunk 4]
```diff
@@ -14,48 +14,68 @@
...
+  val controller = new SDILController(messagesApi, mockSdilConnector) {
+    override def authConnector = mockAuthConnector
+  }
 
   override def beforeEach() {
-    reset(mockSdilConnector)
+    reset(mockSdilConnector, mockAuthConnector)
   }
 
   "HelloWorldController" should {
```
Comment (hmrclt) [inline test/controllers/HelloWorldControllerSpec.scala]: rename this please

File: app/config/frontendAppConfig.scala [hunk 5]
```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.softdrinksindustrylevy.config
+package config
```
Comment (hmrclt) [inline app/config/frontendAppConfig.scala]: could we please use sdil.config instead? And so on and so forth.

Comment (nkaraolis) [reply app/config/frontendAppConfig.scala]: Done

File: project/FrontendBuild.scala (hunk 1 above)
Comment (nkaraolis) [reply project/FrontendBuild.scala]: Fixed

File: test/controllers/HelloWorldControllerSpec.scala (hunk 4 above)
Comment (nkaraolis) [reply test/controllers/HelloWorldControllerSpec.scala]: Changed

File: test/controllers/HelloWorldControllerSpec.scala (hunk 2 above)
Comment (nkaraolis) [reply test/controllers/HelloWorldControllerSpec.scala]: Removed whitespace in tests

File: test/controllers/package.scala (hunk 3 above)
Comment (nkaraolis) [reply test/controllers/package.scala]: Added line break

**Pull request merged**